tests/meta/report/test_rnafusion_api.py

```diff
@@ -33,3 +37,35 @@ def test_get_sample_metadata(
 
     # THEN the sample metadata should be correctly retrieved and match the expected validated metrics
     assert sample_metadata.model_dump() == rnafusion_validated_metrics
+
+
+def test_ensure_rin_thresholds(
```

Comment:
I think more tests with different values of RIN are important, maybe you can parametrise this test